tensorboard/uploader/formatters_test.py

```diff
@@ -27,6 +36,15 @@
 
 
 class TensorBoardExporterTest(tb_test.TestCase):
+    def _run(self, formatter, experiment, experiment_url):
+        """Test helper that ensures formatting is done with fixed timezone."""
+        try:
+            with mock.patch.dict(os.environ, {"TZ": "UTC"}):
+                time.tzset()
+                return formatter.format_experiment(experiment, experiment_url)
```

Comment:
`run` and passing formatter do not semantically correspond. Please name the utility something like `format_experiment` instead?

Reply:
Point taken. I revised it to `_format`. I don't want to be too verbose.